packages/cli/src/lib/defaults/infra-modules/tracer/docker-compose.yaml

```diff
@@ -44,27 +68,80 @@ services:
         condition: service_healthy
 
   frontend:
-    image: signoz/frontend:0.10.0
+    image: signoz/frontend:0.11.0
     container_name: frontend
     restart: on-failure
     depends_on:
+      - alertmanager
       - query-service
     ports:
       - "3301:3301"
     volumes:
-      - ./nginx-config.conf:/etc/nginx/conf.d/default.conf
+      - ../common/nginx-config.conf:/etc/nginx/conf.d/default.conf
```

Comment:
The `../common/nginx-config.conf` file should be copied as well, otherwise it appears as an empty directory in .polywrap/infra/modules and the container fails.
And then this path should be changed back to "./nginx-config.conf:/etc/nginx/conf.d/default.conf"